<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1432</width>
    <height>651</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1432</width>
    <height>651</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1391</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="group4">
       <property name="title">
        <string>Başkanlıklar</string>
       </property>
       <widget class="QTableWidget" name="table4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>431</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="group5">
       <property name="title">
        <string>Kayıtlar</string>
       </property>
       <widget class="QTableWidget" name="table5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>431</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="group3">
       <property name="title">
        <string>Tipler</string>
       </property>
       <widget class="QTableWidget" name="table3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>431</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="group1">
       <property name="title">
        <string>Gemiler</string>
       </property>
       <widget class="QTableWidget" name="table1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>431</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="group2">
       <property name="title">
        <string>Sahipler</string>
       </property>
       <widget class="QTableWidget" name="table2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>431</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>251</width>
          <height>141</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>İşlem Yapılacak Tablo:</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radShip">
              <property name="text">
               <string>Gemi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radOwner">
              <property name="text">
               <string>Sahip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radType">
              <property name="text">
               <string>Tip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radPres">
              <property name="text">
               <string>Başkanlık</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radHistory">
              <property name="text">
               <string>Kayıt</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btnInsertElement">
              <property name="text">
               <string>Eleman Ekle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDeleteElement">
              <property name="text">
               <string>Eleman Sil</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUpdateElement">
              <property name="text">
               <string>Eleman Güncelle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUpdateTable">
              <property name="text">
               <string>Tabloları Güncelle</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>151</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Diğer</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnQuery">
           <property name="text">
            <string>Gemi Sorgusu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTypeOrder">
           <property name="text">
            <string>Tip Sorgusu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commitSql">
           <property name="text">
            <string>Kaydet ve Çık</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1432</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
